os/tools/mkheader.py

```diff
@@ -43,6 +44,7 @@
 kernel_ver = sys.argv[3]
 binary_name = sys.argv[4]
 binary_ver = sys.argv[5]
+binary_ram = sys.argv[6]
```

Comment:
please change 'ram' to 'ram_size', binary_ram_size and BINARY_RAM_SIZE_S

Reply:
I modified.